addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -104,6 +104,31 @@
                 </div>
             </t>
 
+            <div class="before-footer" />
+
+            <div t-if="props.data.pos_qr_code">
+                <br/>
+                <div class="pos-receipt-order-data mb-2">
+                    Need an invoice for your purchase ?
+                </div>
+            </div>
+
+            <div t-if="['qr_code', 'qr_code_and_url'].includes(props.data.headerData.company.point_of_sale_ticket_portal_url_display_mode) and props.data.pos_qr_code" class="mb-2">
+                <img id="posqrcode" t-att-src="props.data.pos_qr_code" class="pos-receipt-logo"/>
+            </div>
+
+            <div t-if="props.data.pos_qr_code">
+                <div class="pos-receipt-order-data">
+                    Unique Code: <t t-esc="props.data.ticket_code"/>
+                </div>
+            </div>
+
+            <div t-if="['url', 'qr_code_and_url'].includes(props.data.headerData.company.point_of_sale_ticket_portal_url_display_mode) and props.data.pos_qr_code">
+                <div class="pos-receipt-order-data" t-attf-class="{{ props.data.ticket_portal_url_display_mode === 'qr_code_and_url' ? 'mt-3' : '' }}">
+                    Portal URL: <t t-esc="getPortalURL()"/>
+                </div>
+            </div>
+
```

Comment:
This is not right. There should be a place where `/pos/ticket` is rendered and you should replace that.

Reply:
Check above lines.